systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2030 @@
+            example:
+              - 0
+              - 0
+          success:
+            type: object
+            description: A boolean or an array of booleans representing succees values for each executed function.
+            example:
+              - true
+              - true
+        required: [success]
+    required: [jid, id, created_timestamp, last_updated_timestamp, state, config]
+
+  ManagedSystemState:
+    title: Managed System State
+    type: string
+    enum:
+      - ActivatedWithoutConnection
+      - Approved
+      - Disconnected
+      - ConnectedRefreshPending
+      - Connected
+      - ConnectedRefreshFailed
+    example: Connected
+    description: >
+      The connection state of a system.
+       * ActivatedWithoutConnection: The system is activated but the connection is not established yet.
+       * Approved: The system is approved but it didn't try to connect yet.
+       * Disconnected: The system is disconnected.
+       * ConnectedRefreshPending: The system is connected and the refresh job is running. A refresh job is sent to the system every time the connection is established in order to gather the system information.
+       * Connected: The system is connected. The refresh job has completed with success.
+       * ConnectedRefreshFailed: The system is connected but the refresh job has failed.
+
+  ManagedSystemActivationData:
+    description: Information about the license of a system
+    type: object
+    properties:
+      activated:
+        type: boolean
+        description: Whether the system is activated or not.
+        example: true
+      license_name:
+        type: string
+        description: The name of the license.
+        example: NILicense
+      license_version:
+        type: string
+        description: The license version.
+        example: 19.0
+    required: [activated]
+
+  ManagedSystemDataProperty:
+    description: Property of a system composed by the last updated timestamp and a dictionary with the actual data.
+    type: object
+    properties:
+      last_updated_timestamp:
+        description: Last updated timestamp.
+        type: string
+        format: iso-date-time
+        example: 2019-02-11T13:32:06.030958Z
+      data:
+        description: The actual data.
+        type: object
+        example:
+          key: value
+    required: [last_updated_timestamp, data]
+
+  ManagedSystem:
+    description: Information about a managed system.
+    title: Managed System
+    type: object
+    properties:
+      id:
+        type: string
+        description: The ID of the system.
+        example: VMware_Virtual_Platform--SN-VMware-56_4d_d2_08_ce_89_1b_70-e6_32_fc_21_6c_d0_d9_f9--MAC-00-0C-29-D0-D9-F9
+      created_timestamp:
+        description: The time when the system has been registered.
+        type: string
+        format: iso-date-time
+        example: 2019-02-19T11:42:25.078Z
```

Comment:
The example for all of the timesamp elements in this definition render as an empty object: {} instead of an ISO-8601 string value.